### PR TITLE
Z1: cleaned up non-required libraries from builds and moved SENSORS d…

### PR DESCRIPTION
**Pull request by alignan**
…eclaration to its own file

### CODE REVIEW

**Comment by alignan** [comment]
:+1:

File: examples/zolertia/z1/Makefile [hunk 1]
```diff
@@ -7,8 +7,11 @@ ZOLERTIA_Z1SP=0
 
 CONTIKI_PROJECT = test-phidgets blink test-adxl345 test-tmp102 test-light-ziglet
 CONTIKI_PROJECT += test-battery test-relay-phidget test-tlc59116 test-sht25
-CONTIKI_SOURCEFILES += sht11.c reed-sensor.c
-APPS=serial-shell
+
+CONTIKI_SOURCEFILES += sht11.c reed-sensor.c sht25.c tlc59116.c light-ziglet.c \
+                       relay-phidget.c sht11.c sht11-sensor.c
```
Comment (greg-king5) [inline examples/zolertia/z1/Makefile]: Do you want to name "`sht11.c`" twice?

**Comment by alignan** [comment]
Oops, thanks!

File: examples/zolertia/z1/Makefile [hunk 2]
```diff
@@ -7,8 +7,11 @@ ZOLERTIA_Z1SP=0
 
 CONTIKI_PROJECT = test-phidgets blink test-adxl345 test-tmp102 test-light-ziglet
 CONTIKI_PROJECT += test-battery test-relay-phidget test-tlc59116 test-sht25
-CONTIKI_SOURCEFILES += sht11.c reed-sensor.c
-APPS=serial-shell
+
+CONTIKI_SOURCEFILES += reed-sensor.c sht25.c tlc59116.c light-ziglet.c \
+                       relay-phidget.c sht11.c sht11-sensor.c
+
+MODULES += dev/sht11
```
Comment (greg-king5) [inline examples/zolertia/z1/Makefile]: I must apologize; when I posted my comment about `sht11.c`, I had not thought, yet, about the meaning of this `MODULES` line.  It compiles the two "sht11*.c" files; therefore, they do not need to be named in the above `CONTIKI_SOURCEFILES` line.

**Pull request merged**